tasks/EGPSR.tex

```diff
@@ -53,44 +39,25 @@ \subsection*{Procedure}
 % %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection*{Additional rules and remarks}
 \begin{enumerate}[nosep]
-	\item \textbf{Partial scoring:} The main task allows partial scoring (per \emph{completed} command).
+	\item \textbf{Partial scoring:} The main task allows partial scoring (per \emph{solved} problem).
 
-	\item \textbf{Command generator:} Tasks will be generated using the official \emph{GPSR Command Generator}\footnote{\url{https://github.com/kyordhel/GPSRCmdGen}}.
-	Commands for EGPSR are either \STWO{} tasks, complex commands requiring to perform chains of subtasks in a sequence, or \emph{incomplete} commands lacking relevant information to succeed.
-
-	\item \textbf{Non-expert operators:} A \emph{Non-Expert Operator} receives a note-card with the summarized command from the referee and rephrases the command.
-	Referees are not allowed to instruct non-expert operators on how to operate the robot.
-	Teams attempting to instruct or bias the operator will be disqualified from the task.
-
-	\item \textbf{Custom operator fallback:} If the robot consistently fails to understand the professional or non-expert operator (three times in a row), the referee will request a custom operator to issue commands.
-
-	\item \textbf{Deus ex Machina:} The scores are reduced if human assistance is received, in particular for:
-	\begin{itemize}
-		\item using a custom operator
-		\item bypassing speech recognition by using an alternative HRI
-		\item receiving human assistance to accomplish a task (score reduction is applied consistent with the other tasks in this rulebook)
-		\item instructing a human assistant to perform the whole task
-	\end{itemize}
+	\item \textbf{Command generator:} Problems and commands will be generated using the official command generator\footnote{\url{https://github.com/johaq/CommandGenerator}}.
 \end{enumerate}
 
 \subsection*{Referee instructions}
 \begin{itemize}
-	\item Provide the note-cards to the operators.
+	\item Setup the problems in the arena.
+	\item Provide commands to volunteers.
 \end{itemize}
 
 \subsection*{OC instructions}
 
 2h before the test:
 \begin{itemize}
-	\item Generate the robot commands (don't reveal them to the teams!).
-	\item Announce the location of the instruction point.
+	\item Generate the problems and commands (don't reveal them to the teams!).
```

Comment:
Is there a reason to allow for 2h possible leak time? Only the main egpsr referee should know them so much in advance. Should be okay, though.

Reply:
Honestly, these things are more of a reminder to set the referee up with everything. In this case make sure each referee has the means to generate commands inside the arena.